README.md

```diff
@@ -1,3 +1,39 @@
-# coSMicQC
+<img height="200" src="https://raw.githubusercontent.com/WayScience/coSMicQC/main/logo/with-text-for-light-bg.png?raw=true">
 
-coSMicQC is a software tool for single-cell quality control of morphology datasets.
+# Single cell Morphology Quality Control
+
+[![Build Status](https://github.com/WayScience/coSMicQC/actions/workflows/run-tests.yml/badge.svg?branch=main)](https://github.com/WayScience/coSMicQC/actions/workflows/run-tests.yml?query=branch%3Amain)
+[![Ruff](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/astral-sh/ruff/main/assets/badge/v2.json)](https://github.com/astral-sh/ruff)
+
+> :stars: Navigate the cosmos of single-cell morphology with confidence — coSMicQC keeps your data on course!
+
+coSMicQC is a software tool to evaluate converted single-cell morphology outputs from CytoTable.
+
+When performing object detection in CellProfiler, it is very hard to get "perfect" compartment segmentation across a large high-throughput screen.
+Technical artifacts can occur during segmentation, including under or over segmentation or segmentation of background or smudges/bright artifacts.
```

Comment:
Consider the following suggestion. Also, it feels like this sentence might flow better after "However..." below.
```suggestion
Technical artifacts can arise during segmentation, leading to issues such as under-segmentation, over-segmentation, or the erroneous segmentation of background noise, smudges, or bright artifacts.
```

Reply:
I think this sounds more clear and I like the use of erroneous! I will accept this change!